client/state/sites/plans/actions.js

```diff
@@ -122,7 +121,7 @@ export function fetchSitePlans( siteId ) {
  * @returns {Object} the corresponding action object
  */
 export function fetchSitePlansCompleted( siteId, data ) {
-	data = reject( data, '_headers' );
+	delete data._headers;
```

Comment:
The problem is that `reject()` doesn't work [as expected](https://github.com/Automattic/wp-calypso/commit/48d61dc7c9cf7d98ce7905ab61320320441cac8f).